src/Map/NumField.jl

```diff
@@ -1063,3 +1063,23 @@ function AbstractAlgebra.show_map_data(io::IO, f::NumFieldHom)
     end
   end
 end
+
+################################################################################
+#
+#  QQ
+#
+################################################################################
+
+struct QQHom{S} <: Map{QQField, S, HeckeMap, Any}
```

Comment:
Huh, should this be in here? Or maybe rename the PR?

Reply:
I'll adjust the commit message